bb-aem-65/src/main/java/com/cognifide/qa/bb/aem/core/component/dialog/dialogfields/DefaultSelect.java

```diff
@@ -26,22 +26,29 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
+import com.cognifide.qa.bb.qualifier.CurrentScope;
 import com.cognifide.qa.bb.qualifier.PageObject;
+import com.cognifide.qa.bb.wait.BobcatWait;
+import com.google.inject.Inject;
 
 /**
  * Default implementation of {@link Select}
  */
-@PageObject(xpath = "//coral-select/..")
+@PageObject(css = Locators.SELECT_CSS)
```

Comment:
Same as above - if this is the only place where the selector is used, leave it here.

Reply:
Fixed